src/lib/apis/objkt/queries.ts

```diff
@@ -0,0 +1,13 @@
+import { gql } from '@apollo/client';
+
+export const buildGetAllUserCollectiblesQuery = (address: string) => {
+  return gql`
+    query MyQuery {
+      token(where: {holders: {holder_address: {_eq: "${address}"}}}) {
+        fa_contract
+        token_id
+        lowest_ask
```

Comment:
you also need `listings_active` for more information about `floorPrice`.
ex.: 
![image](https://github.com/madfish-solutions/templewallet-extension/assets/26839276/3211cc2c-00df-406e-9b72-3050ab217659)

and you need to add all currencies to the project (see mobile repo) and add a display of `floorPrice` in different currencies